src/components/Chart.jsx

```diff
@@ -72,15 +72,17 @@ class Chart extends Component {
             theme,
             position,
             showLastDigitStats,
+            yAxiswidth,
         } = this.props;
 
         const currentPosition = `cq-chart-control-${(chartControlsWidgets && position && !isMobile) ? position : 'bottom'}`;
         const contextWidth =  !isMobile ? `smartcharts-${containerWidth}` : '';
         const TopWidgets = topWidgets || this.defaultTopWidgets;
         const BottomWidgets = !bottomWidgets && showLastDigitStats ? LastDigitStats : bottomWidgets;
+        const HasMarkers = children.length ? 'smartcharts--has-markers' : '';
```

Comment:
if there are any markers, then increase the `subholder` z-index

Reply:
@shayanbinary would you please add this comment to the code?

```suggestion
        // if there are any markers, then increase the subholder z-index
        const HasMarkers = children.length ? 'smartcharts--has-markers' : '';
```